Gemfile

```diff
@@ -2,6 +2,6 @@ source 'https://rubygems.org'
 
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
-gem 'cqm-models', git: 'https://github.com/projecttacoma/cqm-models.git', branch: '_type'
+gem 'cqm-models', git: 'https://github.com/projecttacoma/cqm-models.git'
```

Comment:
If we have the `github` git source above, do we want to use it here?

Reply:
Not critical to getting the PR in, but something that can get done at some point.

Reply:
Yes we definitely want to use the newer style, I've pushed an update.